### PR TITLE
Fall 2023 Semester Launch

### PR DESCRIPTION
**Pull request by AndreasLc1103**
This pull request is the final stage before merging into the production environment.  This contains branches for the development of three different labs. 

- Lab 9: Accessibility to Localization
- Lab 10: Accessibility to Neural Networks
- Lab 11: Accessibility to Literacy

This also contains various other fixes, such as:

- Updated team member headshots
- Refactored older labs (class components to functional components)
- A new alternative to Redux using context (Lab 4, Lab 6, and Lab 8 were refactored to use the context method)

- [x] No discrepancies across browsers. Eg. chrome vs safari
- [x] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [x] CSS (https://jigsaw.w3.org/css-validator/)
- [x] Google sign-in is operational
- [x] Accessibility functions
- [x] Pages can scale without distorting page
- [x] No dead links
- [x] Navbar is consistent across the site
- [x] Pages are screen-reader accessible
- [x] Contrast meets standards for accessibility
- [x] Pages are keyboard accessible
- [x] Passes WAVE Evaluation Tool chrome extension

### CODE REVIEW

File: client/src/components/exercise/lab10/components/ShapeSpawner.js [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+            minDifference = absoluteDifference;
+            closestGapIndex = i;
+          }
+        }
+      }
+
+      if (!_.isEmpty(shapes)) {
+        /* If there is a Shape at every column */
+        if (closestGapIndex === null) {
+          /* Avoid the color with the heaviest weight */
```
Comment (AndreasLc1103) [inline client/src/components/exercise/lab10/components/ShapeSpawner.js:173]: @notcruz need to create an issue for this so it's tracked

**Review by Mster115 — APPROVED** [review]

**Review by AndreasLc1103 — COMMENTED** [review]
looks good to me

**Review by heathermoses — APPROVED** [review]
Lgtm

File: client/src/components/exercise/lab10/components/ShapeSpawner.js (hunk 1 above)
Comment (notjoncruz) [reply client/src/components/exercise/lab10/components/ShapeSpawner.js:173]: will do

**Pull request merged**